### PR TITLE
Fix first subscription message is lost (close #4)

### LINKED ISSUE #4: First message is lost when using subscribe/query method
When we set up a subscription the `_start` method is consuming the first message sent by the subscriber which should not be happening. All the messages should be sent to the callback method passed.

The same applies to the `query` method.

### PR DESCRIPTION
**Pull request by BharathaAravind**
The `_start` was consuming the first reply from the server for both query and subscribe functions.

As per the GraphQL subscription [spec](https://github.com/apollographql/subscriptions-transport-ws/blob/master/PROTOCOL.md), the `GQL_START` message gets a `GQL_DATA` message as a reply for both subscriptions and queries/mutations through websocket. 

This change updates the `_start` and `query` method, where the `_start` no longer waits for the response and returns the `_id` and the first receive would be `GQL_DATA` for the `query` method.

### CODE REVIEW

File: graphql_client/__init__.py [hunk 1]
```diff
@@ -45,11 +45,13 @@ def _conn_init(self, headers=None):
         self._conn.send(json.dumps(payload))
         self._conn.recv()
 
+
```
Comment (ecthiender) [inline graphql_client/__init__.py:48]: remove this extra newline?

File: graphql_client/__init__.py [hunk 2]
```diff
@@ -45,11 +45,13 @@ def _conn_init(self, headers=None):
...
     def _start(self, payload):
         _id = gen_id()
         frame = {'id': _id, 'type': 'start', 'payload': payload}
         self._conn.send(json.dumps(frame))
-        self._conn.recv()
+        # commenting this as the spec excepts data after start and subscription misses the first data if data is received here
+        # res = self._conn.recv()
```
Comment (ecthiender) [inline graphql_client/__init__.py:54]: can you remove the commented lines?

**Review by ecthiender — CHANGES_REQUESTED** [review]

File: graphql_client/__init__.py (hunk 1 above)
Comment (BharathaAravind) [reply graphql_client/__init__.py:48]: @ecthiender I have removed the extra line.

File: graphql_client/__init__.py (hunk 2 above)
Comment (BharathaAravind) [reply graphql_client/__init__.py:54]: I have removed the commented lines.

Comment (BharathaAravind) [reply graphql_client/__init__.py:54]: https://github.com/ecthiender/py-graphql-client/pull/3/commits/e486ccb616fb3286247dce50f14486a784e7444e

File: graphql_client/__init__.py (hunk 1 above)
Comment (BharathaAravind) [reply graphql_client/__init__.py:48]: https://github.com/ecthiender/py-graphql-client/pull/3/commits/e486ccb616fb3286247dce50f14486a784e7444e

**Comment by BharathaAravind** [comment]
I have added a commit with the changes. https://github.com/ecthiender/py-graphql-client/pull/3/commits/e486ccb616fb3286247dce50f14486a784e7444e

**Pull request merged**